workspaces/src/rpc/client.rs

```diff
@@ -50,7 +51,7 @@ use crate::types::{AccountId, InMemorySigner, Nonce, PublicKey};
 use crate::{Network, Worker};
 
 pub(crate) const DEFAULT_CALL_FN_GAS: NearGas = NearGas::from_tgas(10);
-pub(crate) const DEFAULT_CALL_DEPOSIT: Balance = 0;
+pub(crate) const DEFAULT_CALL_DEPOSIT: NearToken = NearToken::from_yoctonear(0);
```

Comment:
I just think this way it is easier to read:

```suggestion
pub(crate) const DEFAULT_CALL_DEPOSIT: NearToken = NearToken::from_near(0);
```